python/cuopt/cuopt/linear_programming/problem.py

```diff
@@ -905,6 +970,17 @@ def get_incumbent_values(self, solution, vars):
             values.append(solution[var.index])
         return values
 
+    def relax(self):
+        """
+        Relax a MIP problem into an LP problem and return the relaxed model.
+        """
+        self.reset_solved_values()
+        relaxed_problem = copy.deepcopy(self)
+        vars = relaxed_problem.getVariables()
+        for v in vars:
+            v.VariableType = CONTINUOUS
+        return relaxed_problem
+
     def post_solve(self, solution):
```

Comment:
`post_solve` has a different meaning in the context of presolver 

here, you are taking a returned output from the solver and placing in the problem class, so may be you can say `copy_solution`

Reply:
updated to change function name